packages/components/src/dimension-control/README.md

```diff
@@ -1,5 +1,9 @@
 # DimensionControl
 
+<div class="callout callout-alert">
+This component is deprecated.
```

Comment:
I'm refraining from issuing any "Use ___ instead" guidance, because nobody uses this anyway (also I don't know 😇).

Reply:
At most we could say "Use `SelectControl` or `CustomSelectControl` with and pass this set of options"

Reply:
Given that the original code example is for padding, I'm guessing most extenders would just use a Dimensions hook now. Doesn't cover the responsive case though 😕